cohort_3/curriculum/maestro-website/README.md

```diff
@@ -0,0 +1,106 @@
+# Maestro + webpage workshop
+
+## Website bootstrap
+
+- Open a cloud9 workspace and create a new folder titled `maestro-website`
+- Inside the folder create a new file called `index.html`
+- Make a basic website template with a `<head>` and `<body>`
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+
+  </head>
+  <body>
+
+</body>
+</html>
+```
+
+- Inbetween `<head>` tags, add `<script>` tags with an attribute `src` that is
+  equal to `"https://maestro.ngrok.com/static/baton.js"`
```

Comment:
Also – do they know how to create a script tag yet? As in how they should also have a closing `</script>` tag?